lang/test/org/partiql/lang/syntax/SqlParserTest.kt

```diff
@@ -3951,6 +3950,44 @@ class SqlParserTest : SqlParserTestBase() {
         )
     }
 
+    //****************************************
+    // OFFSET clause parsing
+    //****************************************
```

Comment:
These are some successful `OFFSET` parsing tests. May want to also add a few `OFFSET` parsing error tests. Consider:
- `OFFSET` with no subsequent argument
- `OFFSET` before `LIMIT`
- etc.